rdr_service/model/bq_participant_summary.py

```diff
@@ -400,6 +400,45 @@ class BQParticipantSummarySchema(BQSchema):
+    enrollment_status_v3_0_participant_time = BQField('enrollment_status_v3_0_participant_time',
+                                                      BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_0_participant_plus_ehr_time = BQField('enrollment_status_v3_0_participant_plus_ehr_time',
+                                                               BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_0_pmb_eligible_time = BQField('enrollment_status_v3_0_pmb_eligible_time',
+                                                       BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_0_core_minus_pm_time = BQField('enrollment_status_v3_0_core_minus_pm_time',
+                                                        BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_0_core_time = BQField('enrollment_status_v3_0_core_time', BQFieldTypeEnum.DATETIME,
+                                               BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1 = BQField('enrollment_status_v3_1', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_id = BQField('enrollment_status_v3_1', BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_participant_time = BQField('enrollment_status_v3_1_participant_time',
+                                                      BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_participant_plus_ehr_time = BQField('enrollment_status_v3_1_participant_plus_ehr_time',
+                                                               BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_participant_plus_basics_time = BQField('enrollment_status_v3_1_participant_plus_basics_time',
+                                                               BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_core_minus_pm_time = BQField('enrollment_status_v3_1_core_minus_pm_time',
+                                                        BQFieldTypeEnum.DATETIME, BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_core_time = BQField('enrollment_status_v3_1_core_time', BQFieldTypeEnum.DATETIME,
+                                               BQFieldModeEnum.NULLABLE)
+    enrollment_status_v3_1_participant_plus_baseline_time = \
+        BQField('enrollment_status_v3_1_participant_plus_baseline_time', BQFieldTypeEnum.DATETIME,
+                BQFieldModeEnum.NULLABLE)
+    health_datastream_sharing_status_v3_1 = BQField('health_datastream_sharing_status_v3_1',
+                                                    BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
+    health_datastream_sharing_status_v3_1_id = BQField('health_datastream_sharing_status_v3_1_id',
+                                                       BQFieldTypeEnum.STRING, BQFieldModeEnum.NULLABLE)
```

Comment:
I'm not sure on this, but should it be INTEGER instead of STRING for the data type on the `health_datastream_sharing_status_v3_1_id` field?

code to clarify what I mean:
```suggestion
                                                       BQFieldTypeEnum.INTEGER, BQFieldModeEnum.NULLABLE)
```

Reply:
Resolved in latest